torch/distributions/utils.py

```diff
@@ -1,16 +1,15 @@
-# mypy: allow-untyped-defs
+from collections.abc import Sequence
 from functools import update_wrapper
-from typing import Any, Callable, Generic, overload, Union
-from typing_extensions import TypeVar
+from typing import Any, Callable, Final, Generic, Optional, overload, TypeVar, Union
```

Comment:
I don't think we are using this, but technically `typing_extensions.TypeVar` has more features than `typing.TypeVar` on older Python versions, hence why PyUpgrade didn't autoupgrade it